src/main/java/com/derongan/minecraft/deeperworld/ecs/SectionChangeListener.kt

```diff
@@ -0,0 +1,21 @@
+package com.derongan.minecraft.deeperworld.ecs
+
+import com.mineinabyss.deeperworld.event.PlayerAscendEvent
+import com.mineinabyss.deeperworld.event.PlayerDescendEvent
+import com.mineinabyss.geary.papermc.access.toGeary
+import org.bukkit.event.EventHandler
+import org.bukkit.event.EventPriority
+import org.bukkit.event.Listener
+
+object SectionChangeListener : Listener {
+
+    @EventHandler(priority = EventPriority.MONITOR)
+    fun PlayerAscendEvent.ascend() {
+        player.toGeary().set(DeeperWorldSection(fromSection.key))
```

Comment:
This is fine for now but we should use our event system now that we have it. ie, set section and add on component change listeners instead of firing events